pkg/command/service/load_test.go

```diff
@@ -733,8 +739,154 @@ func Test_runLoadFromZero(t *testing.T) {
 			LoadConcurrency: "30",
 		}
 		_, _, err = runLoadFromZero(fakeCtx, p, inputs, FakeNamespace, &fakeService)
-		assert.ErrorContains(t, err, "kperf only support hey and wrk now")
+		assert.ErrorContains(t, err, "failed to run external load tool")
 	})
+
+	t.Run("run internal load test tool error", func(t *testing.T) {
+		monkey.Patch(time.Now, func() time.Time {
+			return createTime
+		})
+
+		fakeIngressSvc, err := getFakeIngressService(FakeIngressServiceName, FakeIngressNamespace, false, "", FakeNodePort)
+		if err != nil {
+			return
+		}
+		fakeIngressPod := getFakeIngressPod(FakeIngressServiceName+"fbb76f5df-nzw4c", FakeIngressNamespace, map[string]string{"app": FakeIngressServiceName}, FakeHostIP)
+
+		client := k8sfake.NewSimpleClientset(&fakePodList, &fakeDeployment, &fakeIngressSvc, &fakeIngressPod)
+
+		p := &pkg.PerfParams{
+			ClientSet:        client,
+			NewServingClient: servingClient,
+		}
+		inputs := pkg.LoadArgs{
+			SvcRange:        "0,0",
+			Namespace:       FakeNamespace,
+			SvcPrefix:       FakeServicePrefix,
+			Verbose:         true,
+			Output:          "/tmp",
+			LoadTool:        "default",
+			LoadDuration:    "60hms",
+			LoadConcurrency: "",
+		}
+		_, _, err = runLoadFromZero(fakeCtx, p, inputs, FakeNamespace, &fakeService)
+		assert.ErrorContains(t, err, "failed to run internal load tool")
+	})
+}
+
+func Test_runInternalLoadTool(t *testing.T) {
```

Comment:
```suggestion
func TestRunInternalLoadTool(t *testing.T) {
```

(I think in general test names are usually done in CamelCase, though I noticed there's a bunch with underscores in this file, so maybe there's something I'm overlooking here)

Reply:
Also, and this is me thinking out loud, but would it make sense to have a single test (rather than separate internal / external tool ones) where the load tool was a test parameter? Might help to keep the tests a little cleaner, but could also be a bit overkill... what do you think?

Reply:
Thank you, I will delete underscores in this test file.

Reply:
Good question, these unit tests is just used to test serveral parameter errors, maybe it's more appropriate to test load tool in CI, as load tool need communicate with an valid url.